meeting_notes/20160418_meeting-notes.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Meeting Notes - April 18, 2016
+location: U of T 
+attendees: 10
+date: 2016-04-18
+startTime: '18:30'
+endTime: '21:30'
+---
+
+# Agenda
+
+0. Intros
+1. Vision
+    - Guided with [Brainstorming Gdoc](https://docs.google.com/document/d/1XQiqvkuFiTmaVtZcjmwBUfH5cwZdaYYLXHPqWAB1sB0/edit)
+2. Roadmap
+3. Breakout Groups
+    - Website Content
+    - Maker Proposal
+
+# Notes
+
+- We will use voting in this meeting: 
+    - _Thumbs up_: go ahead, care about the issue and happy with direction
+    - _Closed fist_: "no-block"
+    - _Thumbs down_: needs more discussion, feel strongly enough to block
+
+## Intros
+
+- Went around and everyone gave a brief intro
+    - If people would like to add in their name for the meeting minutes they were asked to add it to the Meeting Agenda Pad
+- Status Update, current approach been two-pronged:
+    - Vision and establishing a direction for the project
+    - Prototyping hardware
+- Want for an open process, documenting as we go, to be available for people looking up information
+- Think about preliminary outreach to other Mesh organizations (Seattle, NYC, Detroit)
+
+## Vision
+
+- Looking at the problem definition doc: https://docs.google.com/document/d/1XQiqvkuFiTmaVtZcjmwBUfH5cwZdaYYLXHPqWAB1sB0/edit
```

Comment:
Can we make these markdown-friendly urls? 
[Brainstorming Gdoc](https://docs.google.com/document/d/1XQiqvkuFiTmaVtZcjmwBUfH5cwZdaYYLXHPqWAB1sB0/edit)